openedx/features/course_experience/tests/views/test_welcome_message.py

```diff
@@ -78,13 +55,17 @@ def setUp(self):
         super(TestWelcomeMessageView, self).setUp()
         self.client.login(username=self.user.username, password=TEST_PASSWORD)
 
+    def tearDown(self):
+        remove_course_updates(self.course)
+        super(TestWelcomeMessageView, self).tearDown()
+
     def test_welcome_message(self):
-        create_welcome_message(self.course, self.user)
+        create_course_update(self.course, self.user, 'First Message', date='January 1 2000')
+        create_course_update(self.course, self.user, 'Second Message', date='January 1 2017')
```

Comment:
It is unclear if the second one is chosen because it has the most recent date, or because the last one is selected, so maybe add a third message with a 2001 date.